app/Http/Controllers/Admin/MonsterCrudController.php

```diff
@@ -91,27 +105,32 @@ public function setupListOperation()
                 'function' => function ($entry) {
                     return 'Created on '.$entry->created_at;
                 },
+                'tab' => 'Function/Closure',
```

Comment:
I think we can move this to miscellaneous or something similar, no need for a tab for this function/closure thing.